report/src/presence_intro.tex

```diff
@@ -0,0 +1,23 @@
+\section{\label{sec:presence_intro}Room Occupancy Detection for Meeting Start and End Times}
+
+To address the problem defined in section \ref{TODO-SECTION 2.a.i AND 2.b.iii}, we propose an occupancy detection system (presence system) to identify when a meeting starts, defined as participants entering a room, and when it ends, defined as all participants leaving the meeting room.
```

Comment:
This will need to be rewritten so it ties well into the prototype evaluation (which has not been written yet)

Reply:
or you could write the prototype evaluation so it ties in well with what has already been written?